src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packageannotation/SuppressionXpathRegressionPackageAnnotation.java

```diff
@@ -0,0 +1,6 @@
+//non-compiled with javac: non-compilable annotation, for testing
+@Deprecated
+package org.checkstyle.suppressionxpathfilter.packageannotation; // warn
```

Comment:
it is kind of strange that this file is not compilable because of annotation. Please share javac output for this file

Reply:
$ javac SuppressionXpathRegressionPackageAnnotation.java   
```
SuppressionXpathRegressionPackageAnnotation.java:2: error: package annotations should be in file package-info.java
@Deprecated
^
1 error
```

Reply:
thanks a lot, please change `for testing` --> `should be in file package-info.java`

Reply:
Doing it right now.

Reply:
Done.